dts/bindings/dma/snps,designware-dma-axi.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2023 Intel Corporation. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Synopsys Designware axi DMA Controller node
+
+compatible: "snps,designware-dma-axi"
+
+include: [dma-controller.yaml, reset-device.yaml]
+
+properties:
+  reg:
+    required: true
+
+  dma-channels:
+    required: true
+
+  interrupts:
+    required: true
+
+  "#dma-cells":
+    const: 1
+
+# #dma-cells : Must be <2>.
```

Comment:
why does this say dma-cells must be 2 but above dma-cells is const 1?

Reply:
Initially design was with two dma-cells. I missed to update the comment properly. I updated now